opencti-platform/opencti-front/src/private/components/settings/exclusion_lists/ExclusionListCreation.tsx

```diff
@@ -86,27 +101,31 @@ const ExclusionListCreationForm: FunctionComponent<ExclusionListCreationFormProp
     });
   };
 
-  const initialValuesFile: ExclusionListCreationFileFormData = {
+  const initialValues: ExclusionListCreationFormData = {
     name: '',
     description: '',
     exclusion_list_entity_types: [],
     file: undefined,
+    content: undefined,
```

Comment:
Prefer using `null` instead of `undefined` in initial values of form.
Why ? When setting undefined you tell React : I don't want you to manage the state of the field. Then we are manipulating the field with React and so it says "wtf you are tricking me changing uncontrolled input to controlled one, don't do that"

Reply:
Done